plugins/package-managers/node/src/funTest/kotlin/NpmFunTest.kt

```diff
@@ -97,12 +96,9 @@ class NpmFunTest : WordSpec({
 
             val result = create("NPM", allowDynamicVersions = true).resolveSingleProject(definitionFile)
 
-            result.issues shouldHaveSize 1
-            with(result.issues.first()) {
-                source shouldBe "NPM"
-                severity shouldBe Severity.ERROR
-                message shouldContain "Unexpected token \"<\" (0x3C) in JSON at position 0 while parsing \"<>\""
-            }
+            result.issues.filter {
+                it.source == "NPM" && it.severity == Severity.ERROR && it.message.contains("Unexpected token")
+            } shouldHaveSize 1
```

Comment:
How about using the `shouldHaveSingleElement { condition }` matcher? Also, prefer the `in` operator over `contains()`.

Reply:
I've done it slightly differently now, but following your underlying idea to use assertions instead of comparisons.